.circleci/config.yml

```diff
@@ -12,6 +12,10 @@ jobs:
           keys:
             - v2-cache-source-{{ .Branch }}-{{ .Revision }}
             - v2-cache-source-{{ .Branch }}
+      - run:
+          name: Git reset
+          command: |
+            git reset --hard
```

Comment:
Solves an unrelated issue with the submodule installation due to caching in circle ci.